vllm/worker/habana_model_runner.py

```diff
@@ -417,6 +417,7 @@ class ModelInputForHPU(ModelRunnerInputBase):
     virtual_engine: int = 0
     lora_mask: Optional[torch.Tensor] = None
     lora_logits_mask: Optional[torch.Tensor] = None
+    async_callback: Optional[Callable] = None
```

Comment:
Where is this set? 
Vllm code names it output_proc_callback_fn shouldnt we keep the name?

Reply:
Formatting issues